addons/modules/stringtable.xml

```diff
@@ -2244,5 +2244,14 @@
             <English>Unable to find suitable position to teleport vehicle to.</English>
             <German>Kein valide Position zum Teleporieren des Fahrzeugs gefunden.</German>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleCrewToGunner">
+            <English>Crew To Gunner</English>
+        </Key>
+        <Key ID="STR_ZEN_Modules_MustHaveGunner">
+            <English>Vehicle must have a primary gunner seat.</English>
+        </Key>
+        <Key ID="STR_ZEN_Modules_NoCrewAvailable">
+            <English>Vehicle must have other crew available.</English>
+        </Key>
```

Comment:
No period at the end for curator feedback messages.

Reply:
https://github.com/zen-mod/ZEN/pull/423/commits/2a92cdddb7ef1cb86dc2564e4b6ed11f1914acb0